rhel/SPECS/grafana.spec

```diff
@@ -104,6 +104,9 @@ getent passwd grafana >/dev/null || \
 exit 0
 
 %changelog
+* Wed Feb 10 2021 Nicola Lamacchia <nicola.lamacchia@percona.com> - 7.1.3-70
```

Comment:
Could you change 70 to 87 as Release variable?

Reply:
Oops, done!